modules/apps/29-fee/ibc_module_test.go

```diff
@@ -611,6 +611,16 @@ func (suite *FeeTestSuite) TestOnAcknowledgementPacket() {
 			},
 			true,
 		},
+		{
+			"success: fee module is disabled, skip fee logic",
+			func() {
+				lockFeeModule(suite.chainA)
+
+				ack = ibcmock.MockAcknowledgement.Acknowledgement()
```

Comment:
This is not true. The ack coming from counterparty will still be incentivized.

We just want to expect that escrow is not paid out